rules/opentitan.bzl

```diff
@@ -166,14 +170,120 @@ def _elf_to_scrambled(ctx):
         )
         return [DefaultInfo(files = depset(outputs), data_runfiles = ctx.runfiles(files = outputs))]
 
-elf_to_scrambled = rule(
-    implementation = _elf_to_scrambled,
+elf_to_scrambled_rom = rule(
+    implementation = _elf_to_scrambled_rom_impl,
     cfg = opentitan_transition,
     attrs = {
         "srcs": attr.label_list(allow_files = True),
         "platform": attr.string(default = OPENTITAN_PLATFORM),
-        "_tool": attr.label(default = "//hw/ip/rom_ctrl/util:scramble_image.py", allow_files = True),
-        "_config": attr.label(default = "//hw/top_earlgrey/data:autogen/top_earlgrey.gen.hjson", allow_files = True),
+        "_tool": attr.label(
+            default = "//hw/ip/rom_ctrl/util:scramble_image.py",
+            allow_files = True,
+        ),
+        "_config": attr.label(
+            default = "//hw/top_earlgrey/data:autogen/top_earlgrey.gen.hjson",
+            allow_files = True,
+        ),
+        "_allowlist_function_transition": attr.label(
+            default = "@bazel_tools//tools/allowlists/function_transition_allowlist",
+        ),
+    },
+)
+
+def _bin_to_flash_vmem_impl(ctx):
+    outputs = []
+    vmem = ctx.actions.declare_file("{}.{}.vmem".format(
+        # Remove ".bin" from file basename.
+        ctx.file.bin.basename[:-4],
+        ctx.attr.word_size,
+    ))
+    outputs.append(vmem)
+    ctx.actions.run(
+        outputs = [vmem],
+        inputs = [
+            ctx.file.bin,
+        ],
+        arguments = [
+            ctx.file.bin.path,
```

Comment:
Did build-cleaner make this into a single list? Do you know if there's a way you can turn off formatting for this list so you can put releated flags on the same line...?

Reply:
Ugh yeah, I wanted to keep related flags on the same line, just like things were in the [meson.build](https://github.com/lowRISC/opentitan/blob/059578e555cb1319577dda5042163406d83368e8/sw/device/meson.build#L49-L59) file, but [it doesn't seem it is possible to disable Buildifier for specific lines](https://github.com/bazelbuild/buildtools/issues/890), though I could be missing something.

Side note: it doesn't seem we run Buildifier in CI yet, so I could check in an "unformatted" BUILD file, but that just kicks the can down the road per se.